app/models/subject_set_import/processor.rb

```diff
@@ -0,0 +1,39 @@
+class SubjectSetImport::Processor
+  attr_reader :subject_set, :uploader
+
+  def initialize(subject_set, uploader)
+    @subject_set = subject_set
+    @uploader = uploader
+  end
+
+  def import(uuid, attributes)
+    subject = subject_set.subjects.where(external_id: uuid).first_or_initialize
```

Comment:
How is the UUID here ensuring unique subject lookups? Or will it clobber subject information if it finds one with an existing `external_id`?

Reply:
Yeah, it'll overwrite. UUID is meant so that an external party doesn't have to keep track of what they've already got uploaded. It's their responsibility to make sure that it's generated consistently. Thinking about it, maybe we can also support CSVs without a UUID column, in which case we just create new subjects for every row.

I should also rename this. In my own Python example notebook I ended up calculating the SHA1 of the images and using that as the "UUID". Panoptes doesn't really care if it's a UUID or some other identifier (hence the `external_id`), so why should the CSV file?